lib/Polaris.Class.ps1

```diff
@@ -1,324 +1,319 @@
+                [Polaris]::Send(
+                    $RawResponse,
+                    $Response.StreamResponse,
+                    $Response.StatusCode,
+                    $Response.ContentType,
+                    $Response.Headers
+                )
+            }
+            else {
+                [Polaris]::Send(
+					$RawResponse,
+					$Response.ByteResponse,
+					$Response.StatusCode,
+					$Response.ContentType,
+					$Response.Headers
+				)
+            }
         }
         catch {
-            throw $_         
+            throw $_
         }
     }
 
-    static [void] Send([System.Net.HttpListenerResponse] $rawResponse, [byte[]]$byteResponse, [int]$statusCode, [string]$contentType, [System.Net.WebHeaderCollection]$Headers) {
-        $rawResponse.StatusCode = $statusCode;
-        $rawResponse.Headers = $Headers;
-        $rawResponse.ContentType = $contentType;
-        $rawResponse.ContentLength64 = $byteResponse.Length;
-        $rawResponse.OutputStream.Write($byteResponse, 0, $byteResponse.Length);
-        $rawResponse.OutputStream.Close();
-    }        
-		
-    static [void] Send(
-        [System.Net.HttpListenerResponse] $rawResponse,
-        [byte[]] $byteResponse,
-        [int] $statusCode,
-        [string] $contentType
-    ) {
-        $rawResponse.StatusCode = $statusCode;
-        $rawResponse.ContentType = $contentType;
-        $rawResponse.ContentLength64 = $byteResponse.Length;
-        $rawResponse.OutputStream.Write($byteResponse, 0, $byteResponse.Length);
-        $rawResponse.OutputStream.Close();
-    }
-
-    [Void] Log([string] $logString) {
-        try {
-            $this.Logger($logString)
-        }
-        catch {
-            Write-Host $_.Message
-            Write-Host $logString
-        }
+    static [void] Send (
+        [System.Net.HttpListenerResponse]$RawResponse, 
+        [byte[]]$ByteResponse, 
+        [int]$StatusCode, 
+        [string]$ContentType, 
+		[System.Net.WebHeaderCollection]$Headers
+	) {
+        $RawResponse.StatusCode = $StatusCode;
+        $RawResponse.Headers = $Headers;
+        $RawResponse.ContentType = $ContentType;
+        $RawResponse.ContentLength64 = $ByteResponse.Length;
+        $RawResponse.OutputStream.Write($ByteResponse, 0, $ByteResponse.Length);
+        $RawResponse.OutputStream.Close();
+	}
+	
+	static [void] Send (
+        [System.Net.HttpListenerResponse]$RawResponse, 
+        [System.IO.Stream]$StreamResponse, 
+        [int]$StatusCode, 
+        [string]$ContentType, 
+		[System.Net.WebHeaderCollection]$Headers
+	) {
+        $RawResponse.StatusCode = $StatusCode;
+        $RawResponse.Headers = $Headers;
+        $RawResponse.ContentType = $ContentType;
+        $StreamResponse.CopyTo($RawResponse.OutputStream);
+        $RawResponse.OutputStream.Close();
     }
 
-
-    Polaris(
-        [Action[string]]$Logger
-    ) {
-        $this.Logger = $Logger
-    }
+	static [void] Send (
```

Comment:
this and the Send on line 252 are really similar (just the difference is `Headers` being included). Can the one without `Headers` just call the other but with `$Headers` set to null?